src/elarm_alarmlist.erl

```diff
@@ -62,7 +62,14 @@ new_alarm(#alarm{ alarm_id = AlId, src = Src, event_id = EventId } = Alarm,
     {ok, State}.
 
 -spec repeat_alarm(alarm(), #al_state{}) -> {ok, #al_state{}} | {error, term()}.
```

Comment:
The spec in line 64 uses a record, please define a type for the record and use that instead.